master/buildbot/www/hooks/github.py

```diff
@@ -170,6 +171,46 @@ def handle_pull_request(self, payload, event):
             len(changes), number))
         return changes, 'git'
 
+    def _get_changes_for_misc_event(self, category, payload):
+        log = Logger()
+        if category == 'create':
+            branch = payload['ref']
+        elif category == 'release':
+            branch = payload['release']['tag_name']
+
+        log.debug('Processing {0},{1}'.format(category, branch))
+        changes = []
+        change = {
+            'when_timestamp': dateparse(payload['repository']['created_at']),
+            'repository': payload['repository']['html_url'],
+            'project': payload['repository']['full_name'],
+            'branch': branch,
```

Comment:
This will fail if category is not `create` or `release`. For the current code that's fine. But maybe you can put in a `else:` case and return an empty list of changes and emit a log message.